integration-libs/cdc/components/login/cdc-user-login.module.ts

```diff
@@ -0,0 +1,33 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, I18nModule, NotAuthGuard, provideConfig, UrlModule } from '@spartacus/core';
+import { FormErrorsModule, SpinnerModule } from '@spartacus/storefront';
+import { LoginFormComponent } from '@spartacus/user/account/components';
+import { facadeProviders } from './facade/facade-providers';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    RouterModule,
+    UrlModule,
+    I18nModule,
+    FormErrorsModule,
+    SpinnerModule,
+  ],
+  providers: [
+    provideConfig(<CmsConfig>{
+      cmsComponents: {
+        ReturningCustomerLoginComponent: {
+          component: LoginFormComponent,
+          guards: [NotAuthGuard],
+          providers: [ ...facadeProviders],
```

Comment:
Could you put the providers directly here?

The term and concept of `facadeProviders` is reserved for providing the concrete Services for the  [Facades (proxy for lazy loading the service's implementation)](https://sap.github.io/spartacus-docs/proxy-facades/#defining-proxy-facades)

Reply:
Noted, thanks